app/src/app.js

```diff
@@ -5,6 +5,32 @@ const app = express();
 const bodyParser = require('body-parser');
 app.use(bodyParser.json());
 
+app.delete('/posts/:id', async (req, res) => {
+    try {
```

Comment:
Wyjątki stosujemy tam gdzie spodziewamy się że coś może się wykrzaczyć, ale opakowanie całego programu w wyjątek to nie jest dobry pomysł. Czasem to że aplikacja się wywali to pożądany efekt bo to oznacza że coś w kodzie należy poprawić